playbook/playbook_ui.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rouge", "~> 3.15"
   s.add_dependency "view_component", "2.55.0"
   s.add_dependency "webpacker-react", "~> 0.3.2"
+  s.add_dependency "will_paginate", "~> 3.3"
```

Comment:
Should this be a dependency? Can't we just tell them they need to bring the Gem in to use it? How do we message Trix? @jasperfurniss @thestephenmarshall

Reply:
Yes. We can make this a dev dependency.

Reply:
Loading the dev dependency may get you past CI, but it means that the application using this Playbook control would be required to load `will_paginate` on its own. At the very least, you'd need to update your instructions for usage.

I'd like for me and Carlos to think over this one a bit though in case we can come up with a better solution.

Reply:
We don't use it, but [kaminari](https://github.com/kaminari/kaminari) is the other popular pagination gem.

Reply:
Sweet! We'll make the `s.add_development_dependency` update just to check CI, and we'll wait to hear from ya'll.